eventhubsprocessor/__init__.py

```diff
@@ -0,0 +1,8 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
In other SDKs we have called it event process host and this name has been accepted by the customers. Consider renaming the folder to "eventprocessorhost"

Reply:
The [C# sdk](https://github.com/Azure/azure-event-hubs-dotnet/tree/dev/src/Microsoft.Azure.EventHubs.Processor) calls it processor and the [java sdk](https://github.com/Azure/azure-event-hubs-java/tree/e6413d059471d75224a62df89251c5dfd331f5e2/azure-eventhubs-eph/src/main/java/com/microsoft/azure/)  calls it eventprocessorhost

Will defer to your preference